python/ql/lib/semmle/python/dataflow/new/internal/DataFlowPrivate.qll

```diff
@@ -564,7 +564,6 @@ predicate neverSkipInPathGraph(Node n) {
  * Holds if `t1` and `t2` are compatible, that is, whether data can flow from
  * a node of type `t1` to a node of type `t2`.
  */
-pragma[inline]
 predicate compatibleTypes(DataFlowType t1, DataFlowType t2) { any() }
```

Comment:
Why has the stub implementation been changed to bind the parameters for some languages (cpp, go) but not others (python, swift)?

Reply:
Because in python and swift `DataFlowType` is a singleton.

Reply:
I know that Go doesn't use types in dataflow, but I forgot that we have a hack in place to avoid an optimiser bug which means `DataFlowType` isn't a singleton. Apparently I can [remove it now](https://github.com/github/codeql/pull/16819).